frontend/pages/schedule/ManageSchedulePage/ManageSchedulePage.tsx

```diff
@@ -238,9 +244,36 @@ const ManageSchedulePage = ({
     handleTeamSelect(teams[0].id);
   }
 
+  // If team_id from URL query params is not valid, we instead use a default team
+  // either the current team (if any) or all teams (for global users) or
+  // the first available team (for non-global users)
```

Comment:
I'm not sure this comment is necessary as the code below is nicely readable.

Reply:
I copied comments directly from ManagePoliciesPage

Reply:
Got it. I'm working on cutting down on superflous comments in the code. If you feel these comments are necessary to make the code easily readable, feel free to leave them in. If not, please remove them.

Reply:
I think we should leave it until someone addresses the tech debt between ManageSchedules and ManagePolicies I filed on #3942

Reply:
Works for me!